packages/console/package.json

```diff
@@ -21,20 +25,24 @@
     "url": "https://github.com/patternfly/patternfly-react/issues"
   },
   "homepage": "https://github.com/patternfly/patternfly-react/blob/master/src/console/README.md",
-  "publishConfig": {
-    "access": "public"
+  "sassIncludes": {
+    "patternfly": "../../node_modules/patternfly/dist/sass",
+    "bootstrap": "../../node_modules/bootstrap-sass/assets/stylesheets",
+    "fontAwesome": "../../node_modules/font-awesome-sass/assets/stylesheets"
   },
   "scripts": {
-    "build": "node ../../scripts/runScript.js build:console",
-    "prepublish": "yarn build"
+    "prebuild": "yarn build:scripts && yarn build:sass && yarn build:less",
+    "build:scripts": "babel src --out-dir dist/js --ignore .test.js,__mocks__",
+    "build:less": "mkdir -p dist/less && cp -r less/* dist/less",
+    "build:sass": "mkdir -p dist/sass && cp -r sass/* dist/sass && node-sass --output-style compressed --include-path sass --include-path $npm_package_sassIncludes_patternfly --include-path $npm_package_sassIncludes_bootstrap --include-path $npm_package_sassIncludes_fontAwesome -o dist/css sass/console.scss"
```

Comment:
These could probably live in the root too.  It would be nice to have it build similar to babel if possible.

Reply:
i agree w/ this, but i'm not quite sure how to accomplish it since we have package specific [file].scss at the end... feel free to give this a go...but i don't think it's really a blocker.